File: ObjectRequestBrokerJava/src/main/java/ORB/namingservice/requests/LocalizationRequest.java
package ORB.namingservice.requests;

public class LocalizationRequest extends  Request{
    private String entry_name;

    public LocalizationRequest(String entry_name) {
        super("localization_request");
        this.entry_name = entry_name;
    }

    public LocalizationRequest() {
    }

    public String getEntry_name() {
        return entry_name;
    }

    public void setEntry_name(String entry_name) {
        this.entry_name = entry_name;
    }


}

File: ObjectRequestBrokerCS/ORB/namingservice/requests/LocalizationRequest.cs
using namingservice.requests;

namespace ORB.namingservice.requests
{
	public class LocalizationRequest : Request
	{
		private string entry_name;

		public LocalizationRequest(string entry_name) : base("localization_request")
		{
			this.entry_name = entry_name;
		}

		public LocalizationRequest()
		{
		}

		public virtual string EntryName
		{
			get
			{
				return entry_name;
			}
			set
			{
				entry_name = value;
			}
		}



	}

}